test/framework/cluster.go

```diff
@@ -1140,6 +1147,15 @@ func (e *ClusterE2ETest) SetPackageBundleActive() {
 	}
 }
 
+// ValidatingNoPackageController make sure there is no package controller.
+func (e *ClusterE2ETest) ValidatingNoPackageController() {
+	kubeconfig := e.kubeconfigFilePath()
+	_, err := e.KubectlClient.GetPackageBundleController(context.Background(), kubeconfig, e.ClusterName)
```

Comment:
Not sure how this fulfills the task to confirm there is no package controller. Will the existence of package controller always lead to an error state?

Reply:
It checks if err == nil. Which means it was able to get a packagebundlecontroller.

Reply:
The helm chart creates a PBC